README.md

```diff
@@ -523,55 +522,91 @@ Judgement is completed when the Judge has decided all questions and is not overr
-	XX is the number representing how often this player has been on Trial.
+rule, a procedure defined as a _Trial_ can take place. A Trial is concluded with a _Verdict_
+that will be enacted by someone who has this power.
+
+**359** *Roleplaying in Court*
+
+A Trial features three roles:
+
+- A defendant, the player charged with the violation.
+- A prosecutor.
+- A jury, comprised of at least one player.
+
+No player can take multiple roles in a single Trial, i.e. a player can not be both prosecutor and jury.
+
+**363** *Jury Selection*
+
+The jury in a Trial is selected in a procedure called _Jury Selection_. Jury Selection is defined as follows.
+Both prosecution and defendant must call forth at least one player for jury duty. (Note: There is no uniqueness
+requirement, which means a one-manned jury is still a valid jury.)
+
+**362** *Perfect Prosecutor*
+
+The prosecutor of a Trial is selected as follows:
+
 - If a player accuses another player of breaking a rule, the accuser will take the role of prosecutor in the Trial.
 - Else If no other method of choosing a prosecutor in a Trial is defined, the prosecutor will be selected through the
 	procedure for invoking Judgement. In this procedure the defendant will take the role of the person invoking Judgement.
-- The prosecutor will outline the actions of the defendant that he claims are in violation of the rules and demand a
+
+**361** *The Court Room*
+
+A Trial shall take place in a GitHub issue with the Trial label. The Trial shall be named "The people vs #githubhandle
+XX" where #githubhandle is the name of the defendant and XX is the number representing how often this player has been on
+Trial.
+
+**360** *The Trial Procedure*
+
+A Trial procedure takes place in the following way:
+
+1. The roles are assigned through their respective processes.
+2. The prosecutor will outline the actions of the defendant that he claims are in violation of the rules and demand a
 	punishment in the form of a non-negative integer point reduction of the defendants point. This reduction can be of 0
 	points.
-- The defendant is allowed to defend his actions and can plead either _Guilty_ or _Not Guilty_. If he pleads Guilty, he
+3. The defendant is allowed to defend his actions and can plead either _Guilty_ or _Not Guilty_. If he pleads Guilty, he
 	can propose an alternative punishment in the form of a non-negative integer point reduction no larger than that of the
 	prosecution. If he pleads Not Guilty, an alternative punishment of a zero point reduction is assumed.
-- The prosecution and jury may ask the defendant about his actions and the defendant should either answer these
+4. The prosecution and jury may ask the defendant about his actions and the defendant should either answer these
 	questions to the best of his abilities, or exercise his right to remain silent. Lying during a Trial is an
 	offense for which a player can be put on Trial.
```

Comment:
Hmm, well let's take it one change at a time maybe? We can fix that one in a later PR. I will already be extending my chain with some comments from you guys, but this one I still would like to keep in. Hence I propose leaving it as is for this chain and we can look at it in a new rule-change when this chain is through.